app/src/components/main/analytics/utils.js

```diff
@@ -34,6 +34,34 @@ export const provideEcGA = ({
   ga(command, hitType, eventName, action);
 };
 
+export const provideBulkEcGA = ({
+  name,
+  data,
+  action,
+  command = 'send',
+  hitType = 'event',
+  eventName = 'ecommerce',
+}) => {
+  const ga = ReactGA.ga();
+
+  data.forEach((el) => {
+    ga(`ec:${name}`, el);
+  });
+  ga(command, hitType, eventName, action);
+};
+
 export const normalizeDimensionValue = (value) => {
   return value !== undefined ? value.toString() : undefined;
 };
+
+export const formatEcDashboardData = (dashboard) => {
+  return dashboard.widgets.map((widget, index) => {
+    return {
+      id: widget.widgetId,
+      name: widget.widgetName,
```

Comment:
Name should contain the `widgetType` in human readable format, so the `statisticTrend` will be `Launch Statistics Chart`,  not `widgetName` that specified by user (see the screenshot below).

![image](https://user-images.githubusercontent.com/31861805/147108561-3aa9be85-1f75-4f08-b38b-ab7ad21d1a2f.png)